cerberus-web/src/main/java/com/nike/cerberus/error/SfxAwareApiExceptionHandlerUtils.java

```diff
@@ -81,4 +89,15 @@ public String buildErrorMessageForLogs(
           .inc();
     }
   }
+
+  private void reactSensitiveHeaders(RequestInfoForLogging request) {
```

Comment:
Should `reactSensitiveHeaders` be `redactSensitiveHeaders`?

Reply:
Doh